File: Programming with C/C Programs/SelfLearningPrograms/PassOrFail.c
#include<stdio.h>
int main()
{
    int Physics, Chemistry, Maths, Computer;
    // Maximum marks in each subjest is 25
    printf("Enter your marks in Physics\n");
    scanf("%d", &Physics);
    printf("Enter your makrs in Chemistry\n");
    scanf("%d", &Chemistry);
    printf("Enter your marks in Maths\n");
    scanf("%d", &Maths);
    printf("Enter your marks in Computer\n");
    scanf("%d", &Computer);
    printf("Your Total Marks will be %d\n", Physics + Chemistry + Maths + Computer);
    printf("Your percentage is %d\n", (Physics + Chemistry + Maths + Computer) );
    if(Physics<=15 || Chemistry<=15 || Maths<=15 || Computer<=15 || Physics + Chemistry + Maths + Computer<= 40 )
    {
        printf("You are fail\n");
    }
    else 
    printf("You are pass");
    return 0;
    
}

File: FirstJavaProject/src/com/company/pass_or_fail.java
package com.company;

import java.util.Scanner;

public class pass_or_fail {
    public static void main(String[] args) {
        byte physics, chemistry, maths;
        short result;
        float percentage;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the marks of students in Physics, Chemistry and Maths respectively");
        System.out.println("Only Enter the marks out of 100 in each subject");
        physics = sc.nextByte();
        chemistry = sc.nextByte();
        maths = sc.nextByte();
        result = (short) (physics + chemistry + maths);
        percentage = (result / 300f) * 100;
        System.out.println("Your percentage is: " + percentage);
        if (physics >= 33 && chemistry >= 33 && maths >= 33 && percentage >= 40) {
            System.out.println("You have successfully cleared the exams, Congrats");
        } else {
            System.out.println("You didn't make it this year, Sorry! Work harder next time");
        }
    }
}
